slingshot/marc.py

```diff
@@ -63,7 +64,7 @@ def __next__(self):
         while True:
             idx = self.__buffer.find(RECORD_TERMINATOR)
             if idx >= 0:
-                chunk = self.__buffer[:idx]
+                chunk = self.__buffer[:idx+1]
```

Comment:
Confirming this is intentional? It feels like it could be an unintentional change unless the old behavior has been confirmed to be broken.

Reply:
Yeah it's intentional. The record terminator should be included (pretty sure the aleph records didn't always do this correctly which is why it was originally not).

Reply:
Gotcha. Thanks.